ui/component/layoutWrapperText/view.jsx

```diff
@@ -20,14 +20,19 @@ type Props = {
   nsfw: boolean,
   claim: StreamClaim,
   thumbnail: ?string,
+  contentType: string,
+  fileType: string,
 };
 
 function LayoutWrapperText(props: Props) {
-  const { uri, claim, title, nsfw } = props;
+  const { uri, claim, title, nsfw, contentType, fileType } = props;
+
+  const markdownType = ['md', 'markdown'];
+  const isMd = markdownType.includes(fileType) || contentType === 'text/markdown' || contentType === 'text/md';
 
   return (
     <div>
-      <div className="main__document-wrapper">
+      <div className={`main__document-wrapper${isMd ? '-md' : ''}`}>
```

Comment:
Generally the pattern we have been following is this:
```
<div className={classNames('main__document-wrapper', { 'main__document-wrapper--markdown': isMd })}>

```

Where `main__document-wrapper` is the default style, then add any overrides to the class with `--markdown`.

Reply:
Thanks